src/System.Transactions/src/System/Transactions/CommittableTransaction.cs

```diff
@@ -36,18 +35,20 @@ internal CommittableTransaction(IsolationLevel isoLevel, TimeSpan timeout) : bas
             // fill in the traceIdentifier field here.
             _internalTransaction._cloneCount = 1;
             _cloneId = 1;
-            if (DiagnosticTrace.Information)
+            TransactionsEtwProvider etwLog = TransactionsEtwProvider.Log;
+            if (etwLog.IsEnabled())
             {
-                TransactionCreatedTraceRecord.Trace(SR.TraceSourceLtm, TransactionTraceId);
+                etwLog.TransactionCreated(this, "CommittableTransaction");
```

Comment:
You might consider taking advantage of some newer C# features, like [CallerMemberName], [CallerLineNumber], etc.  I recently did so in https://github.com/dotnet/corefx/pull/13134 if you're looking for an example.